web/packages/teleport/src/Discover/Database/CreateDatabase/useCreateDatabase.ts

```diff
@@ -192,6 +214,53 @@ export function useCreateDatabase() {
     setPollActive(true);
   }
 
+  // attemptDbServerQueryAndBuildErrMsg tests if the duplicated `dbName`
+  // (determined by an error returned from the initial register db attempt)
+  // is already a part of the cluster by querying for its db server.
+  // This is an attempt to provide accurate actionable steps for the
+  // user.
+  async function attemptDbServerQueryAndBuildErrMsg(
+    dbName: string,
+    isAwsRds = false
+  ) {
+    const preErrMsg = 'failed to register database: ';
+    try {
+      await ctx.databaseService.fetchDatabase(clusterId, dbName);
+      let message = `a database with the name "${dbName}" is already \
+      a part of this cluster, `;
+      if (isAwsRds) {
+        message += `set "teleport.dev/database_name" tag on this RDS instance to `;
+      }
+      message += `use a different name and try again`;
```

Comment:
I think this might read better as 

> change the value of the "teleport.dev/database_name" tag on the RDS instance to a different name and try again